# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xf86-video-impact/hotfix-wait-return.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-impact/src/impact_driver.c.vanilla	2024-04-09 23:25:03.051797285 +0200
+++ xf86-video-impact/src/impact_driver.c	2024-04-09 23:26:37.336796872 +0200
@@ -224,7 +224,6 @@
 	Bool foundScreen = FALSE;
 	GDevPtr *devSections;
 	GDevPtr dev = 0;
-	resRange range[] = { {ResExcMemBlock,0,0}, _END };
 	struct probed_id probedIDs[IMPACT_MAX_BOARDS];
 	memType base;
 
--- xf86-video-impact/src/impact.h.vanilla	2024-04-09 23:27:46.699796567 +0200
+++ xf86-video-impact/src/impact.h	2024-04-09 23:30:56.895795733 +0200
@@ -84,9 +84,9 @@
 
 	/* impact register handling for FullHouse/SpeedRacer: */
 	void (*WaitCfifoEmpty)(ImpactRegsPtr);
-	unsigned (*WaitCfifo)(ImpactRegsPtr);
-	unsigned (*WaitDMAOver)(ImpactRegsPtr);
-	unsigned (*WaitDMAReady)(ImpactRegsPtr);
+	void (*WaitCfifo)(ImpactRegsPtr);
+	void (*WaitDMAOver)(ImpactRegsPtr);
+	void (*WaitDMAReady)(ImpactRegsPtr);
 	unsigned short (*Vc3Get)(ImpactRegsPtr, CARD8);
 	void (*Vc3Set)(ImpactRegsPtr, CARD8, unsigned short);
 	CARD32 (*XmapGetModeRegister)(ImpactRegsPtr, CARD8);
